packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -173,11 +175,17 @@ function InstalledFonts() {
 function Footer( { shouldDisplayDeleteButton, handleUninstallClick } ) {
 	const { saveFontFamilies, fontFamiliesHasChanges, isInstalling } =
 		useContext( FontLibraryContext );
+
+	const canUserDelete = useSelect( ( select ) => {
+		const { canUser } = select( coreStore );
+		return canUser( 'delete', 'font-families' );
+	} );
+
```

Comment:
I'm sure it's my mistake but this is always returning false and preventing the button from showing.

Reply:
Yup, this was my mistake. ;)